templates/repo/view.tmpl

```diff
@@ -1,4 +1,8 @@
 {{template "base/head" .}}
+<script type="module">
+window.config.pageData.folderIcon = {{ctx.RenderUtils.RenderFolderIcon false}};
+window.config.pageData.openFolderIcon = {{ctx.RenderUtils.RenderFolderIcon true}};
+</script>
```

Comment:
Why .........

Reply:
I know you are trying to "unify the icon sources"

But why inline scripts?

Reply:
I do not think it is right.

You need to provide the open/closed icons for each entry item

Reply:
Currently, this is truly the only implementation method that comes to my mind. 😅